.gitignore

```diff
@@ -103,3 +103,6 @@ packages/amplify-category-notifications/lib
 packages/amplify-category-interactions/lib/
 packages/amplify-environment-parameters/lib/
 !packages/amplify-graphql-model-transformer/src/__tests__/overrides/build/override.js
+
+.nx/cache
+.nx/workspace-data
```

Comment:
nit: could you add new line at the end of of this file and prettier ignore as well ? (why -  https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline )

there should a way to configure IDE to do this automatically, for example https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files .